### DISCUSSION TITLE
emqttd 2.3.11 process exit.

### DISCUSSION THREAD

**Discussion by pinpindao** [emqx/emqx#8569] [General]
Nearly two days, emqttd process exit twice, 1883 and 18083 port cannot connnect, no any logs generate in log directory.
Who have any experiences related this problem?
Thanks!

OS: Windows Server 2012 R2
emqttd: 2.3.11

**Comment by lafirest** [comment]
@pinpindao Hi,
This version is too old, and it has exited maintenance and support a long time ago. It is recommended that you switch to EMQX 4.4 or 5.0
